### DISCUSSION TITLE
Assigning IP address to VM interface gives error.

### DISCUSSION THREAD

**Discussion by tbmay** [netbox-community/netbox#13703] [Q&A]
Hello.  After assigning an IP address to a virtual interface I got the error below.  I didn't have the issue with 3.5.x.  Does anyone have any insight.  Thanks a million.

<class 'AttributeError'>

'VirtualMachine' object has no attribute 'oob_ip_id'

Python version: 3.10.12
NetBox version: 3.6.0
Plugins: None installed

**Comment by tbmay** [comment]
Additional note.  This is happening when I click on every IP address that is attached to a VM.

**Comment by candlerb** [comment]
Already reported as #13619, and fixed in #13621.
